src/tic_tac_toe/core.clj

```diff
@@ -4,21 +4,23 @@
             [tic-tac-toe.game :as game]
             [tic-tac-toe.io :as io]
             [tic-tac-toe.player :as player]
+            [tic-tac-toe.setup :as setup]
             [tic-tac-toe.user-interface :as ui]))
 
 (defn play
-  [game-ui player1 board]
+  [game-ui current opponent board]
   (loop [game-ui game-ui
-         player1 player1
+         current current
+         opponent opponent
          board board]
     (if (board/full? board)
       (ui/prompt-gameover game-ui board)
-      (recur game-ui player1 (game/take-turn* game-ui player1 board)))))
+      (recur game-ui opponent current (game/take-turn* game-ui current board)))))
```

Comment:
so following through with the take-turn*, does this mean if a move is invalid, it still switches the players?

Reply:
I believe it actually breaks.  I pulled out proper validation into an entirely different story, and perhaps that wasn't a fantastic idea.  I can go ahead and see about including it.

Reply:
No it's ok, we can talk about adding acceptance criteria to the stories moving forward in the next sprint

Reply:
OK, sounds good.